version/info.go

```diff
@@ -31,6 +31,7 @@ var (
 	BuildUser string
 	BuildDate string
 	GoVersion = runtime.Version()
+	userAgent = fmt.Sprintf("Prometheus/%s", Version)
```

Comment:
I'm not quite sure what you mean. Which other settings are determined automatically?

Reply:
The above settings like build user

Reply:
Oh. Those are injected at build time, right? So The idea would be to do the same through the build/make commands?

I would imagine it would work something like this then:

```go
var Product string
var userAgent = fmt.Sprintf("%s/%s", Product, Version)
```

Reply:
Perhaps default `Product` to `prometheus-common` in case people don't set the flag during build?

Reply:
Sounds reasonable

Reply:
I've updated the code and commit message to reflect the changes.